runtimevar/runtimevar.go

```diff
@@ -495,16 +497,16 @@ func BytesDecode(b []byte, obj interface{}) error {
 //
 // post defaults to BytesDecode. An optional decoder can be passed in to do
 // further decode operation based on the decrypted message.
-func DecryptDecode(ctx context.Context, k *secrets.Keeper, post Decode) Decode {
-	return func(b []byte, obj interface{}) error {
+func DecryptDecode(unusedCtx context.Context, k *secrets.Keeper, post Decode) Decode {
```

Comment:
Remove this Context parameter: it was added specifically to curry the context, which is what led me to file #1667.

```suggestion
func DecryptDecode(k *secrets.Keeper, post Decode) Decode {
```

Reply:
Thanks, will do!